shared/cfg/machines.cfg

```diff
@@ -16,6 +16,14 @@ variants:
             cd_format = ahci
         enable_pvpanic = yes
         ioport_pvpanic = 0x505
+        pci_controllers = root_port pcie_switch dmi2pci_bridge pci_bridge
+        pci_bus = pci.0
```

Comment:
I'm wondering whether it would be better to set everything to the `pcie_switch` and only change it when explicitly stated (like for nics, ...).

And if we think that most devices should for compatibility issues go to `PCI`, then we should probably set this to `pci_bridge` instead, what do you think?

I tried the behavior on the `virtio_console.spread_linear.specifiable.virtserialport.virtio_console_smoke.open` test, which by default does not define the `virtio_port`, therefor the ports end up in `pcie.0`. When using this (after fixing the https://github.com/avocado-framework/avocado-vt/pull/657/commits/fde21e57f0db8b418821b3d881df41577a290538#r74894177 ) it puts them into the proper switch/bridge instead by default, which is what I'd expect.